llvm/lib/Target/AMDGPU/AMDGPURegisterBankInfo.h

```diff
@@ -165,7 +165,7 @@ class AMDGPURegisterBankInfo final : public AMDGPUGenRegisterBankInfo {
   bool isDivergentRegBank(const RegisterBank *RB) const override;
 
   unsigned copyCost(const RegisterBank &A, const RegisterBank &B,
-                    unsigned Size) const override;
```

Comment:
Can you pre-commit all of these unsigned->TypeSize pieces?

Reply:
I have them separated into their own commit. It is the first commit in the commit history of this Pr. I will commit it separate.